src/sbt-test/sbt-1.5/cross-build-scala3/.scalafix-3.conf

```diff
@@ -0,0 +1 @@
+rules = [OrganizeImports]
```

Comment:
this is not detected as a rename, because I removed the unexercised `DisableSyntax`